features/admin/locale/channel_independent_translatable_entities_locales.feature

```diff
@@ -10,7 +10,7 @@ Feature: Channel independent translatable entities locales
         And it uses the "English (United States)" locale by default
         And I am logged in as an administrator
 
-    @todo @ui
+    @no-api @todo @ui
```

Comment:
```suggestion
    @no-api @ui
```
🤔

Reply:
`api-platform-3` bases on 2.0 with BAP, so this scenario is not yet implemented